server/api/expense-pool/index.js

```diff
@@ -18,6 +18,14 @@ router.get('/:id', isPoolMember(), controller.getPool);
  */
 router.post('/:id/users', isPoolMember(), controller.addUserToPool);
 
+// ------------ Routes above have updated controllers ----------------
+
+// GET => show all pools
+// Added for testing only
+router.get('/test', controller.getPools);
```

Comment:
You got it. I think these got added back into your version when I did the `pull --rebase`
Adding the conditional wrapper to the test route now.